### ISSUE TITLE
v3.0.0 - Selection printing issues

### ISSUE THREAD

**Issue by cleidigh** [cleidigh/printing-tools-ng#228]
@madvinegar
@Mark3032
Better to have separate issues and leave the Release status alone.

Summary of issues : 
- see tail end of : 
- #225 
- Pagination issues where headers are printed on separate pages
- Check if signatures are issue 
- Check if simplified mode issue

**Comment by cleidigh** [comment]
@madvinegar
@Mark3032
I am going to need some specific steps to reproduce and ideally an example email 
So far I cannot reproduce with multi page email, selection or no selection 
I also tried with a signature 

@cleidigh

**Comment by madvinegar** [comment]
The main thing to keep in mind is that when you select text from the email body and then click on the "NG Print" button, it separates the header and the main email body. The header remains in the first page, and the main email body is moved to the second page (for no reason). So the first page shows just the header and the second page shows just the main email body.

The only way to fix this (and print the header and the main email body in the same first page) is first to go to "view" > "show Main body of the message as" > and select to show the email in "Simplified HTML" and then click on the "NG Print" button to print out the email. If you do that, it works correctly.

If you want to revert the looks of the emails as they were before, you need to change their format back to "Original HTML".

**Comment by cleidigh** [comment]
@madvinegar
Still no luck, original or simplified print correctly 
Either my test emails are in a different format then yours or we have some settings different.
@cleidigh

**Comment by Mark3032** [comment]
Do you have an e-mail address where a test e-mail can be sent to you so that you don't have to post it here publicly with your private data and private e-mail address?

**Comment by cleidigh** [comment]
@Mark3032
@madvinegar
Yes that would be great.
Use my test account : 

test1@kokkini.net

@cleidigh

**Comment by Mark3032** [comment]
@cleidigh 
Ich habe Dir zwei e Mails auf Deine eMail Adresse weitergeleitet, wo das Problem auftritt (Header auf der 1. Seite, Text auf der 2. Seite). Ich hoffe, damit lässt es sich reproduzieren.

**Comment by cleidigh** [comment]
@Mark3032 
Danke! I am now able to see the effect of original versus simplified.

<img width="960" alt="2023-07-26-14-55-26 " src="https://github.com/cleidigh/printing-tools-ng/assets/25272315/e15ec0e2-e2e4-4fed-a5fa-d7d4478959c2">


BUT, guess what the screenshot below is... Same email in original view mode - From Thunderbird native print!!
So this issue is independent of PTNG. Try this yourself.

<img width="960" alt="2023-07-26-14-58-42 " src="https://github.com/cleidigh/printing-tools-ng/assets/25272315/223700ec-2317-4c70-88a7-01eb1555b69d">

@cleidigh

**Comment by Mark3032** [comment]
@cleidigh 

Ja, das stimmt, das Problem tritt auch im normalen Druckmodus von Thunderbird auf - unabhängig von Printing Tolls NG. 

Das Problem hat ein anderer User aus  dem Thunderbird deutschsprachigen Thunderbird Forum hier eingemeldet:
https://bugzilla.mozilla.org/show_bug.cgi?id=1843628

**Comment by cleidigh** [comment]
@Mark3032 
That's it..Problem is Outlook.
So we will have to wait for a patch.
I am going to change the issue title since this has nothing to do with selections.
Christopher

**Comment by cleidigh** [comment]
@Mark3032 
@madvinegar 
So here are the conclusions : 
- The page break after the headers is not related to any selection 
- The problem occurs with native Thunderbird printing independent of PTNG 
- The problem occurs with emails from Outlook see
- https://bugzilla.mozilla.org/show_bug.cgi?id=1843628
- We await a patch

@cleidigh

**Comment by madvinegar** [comment]
Ok, so to clarify this, we get this problem only if the person sending the email to us is using outlook.
And we will have to wait for an update from the thunderbird team.

**Comment by cleidigh** [comment]
@madvinegar 
Answering clarifications first:
- So far the reports of multiple users indicate that Outlook is the problem, our test emails seem to also be Outlook 
- There is the Thunderbird bug above we have to wait for
- A partial remedy as you found is to use simplified mode

On your other observations:
- The window global print shortcut Control-P is not yet working in b11
- In upcoming b12 I have this working 
- I think the File/Print menu has a menu shortcut that is not overriden, we may have to live with that 
- Your were invoking the native print, therefore you don't get any ptng formatting 

Does that make sense?
